index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>Mourid Portfolio</title>
+</head>
+<body>
+    <section>
+    <nav class="navbar">
+        <div class="container">
+            <div class="logo"><img src="img/My Logo.png" alt=""></img></div>
+        <div class="menu"><img src="img/Union.svg" alt=""></div>
```

Comment:
- [ ] Please the `nav` tag presents navigational links, so it is not semantic to put the logo in it, after you change the location of the logo, please use a link to create the logo itself because if you look at most of the websites the logo always sends you to the homepage.